Sources/src/FromSTMTestBeamData_source.cc

```diff
@@ -100,14 +100,14 @@ namespace mu2e {
   STMTestBeamDataDetail::STMTestBeamDataDetail(const Parameters& conf,
       art::ProductRegistryHelper& rh,
       const art::SourceHelper& pm)
-    : myModuleLabel_("FromSTMTestBeamData")
+    : myModuleType_(conf().module_type())
```

Comment:
Ah.  I was mislead by the variable name.  Since it's really myModuleType, it's perfectly OK to hard code the string literal here.  Your call on whether to leave it as is or change it back.

Reply:
Thanks, Rob. I will leave it as it is for now